web/_js/atlas.js

```diff
@@ -510,7 +510,7 @@ var atlas = [
 { "id": 459, "submitted_by": "BigPortal", "name": "Wavetro", "description": "Two characters from ex-YouTuber Wavetro's final series, 'the untitled Teddy and Ben show'. The fish Teddy is in the background and the stickman Ben is in the front. ", "website": "https://www.youtube.com/channel/UCyFcxmkcQernyo0PmddhtGA", "subreddit": "r/wavetro", "center": [ 1056.5, 407.5 ], "path": [ [ 1062.5, 413.5 ], [ 1062.5, 402.5 ], [ 1061.5, 401.5 ], [ 1050.5, 401.5 ], [ 1050.5, 413.5 ], [ 1062.5, 413.5 ] ]}, 
 { "id": 462, "submitted_by": "Beegrene", "name": "Octavia", "description": "Octavia is a minor character from My Little Pony Friendship is Magic. She is often seen playing various stringed instruments. The '2SET' in her mane refers to TwoSet Violin, a comedy duo.", "website": "", "subreddit": "/r/mylittlepony", "center": [ 1343.5, 1985.5 ], "path": [ [ 1316.5, 1973.5 ], [ 1370.5, 1973.5 ], [ 1370.5, 1997.5 ], [ 1316.5, 1997.5 ], [ 1316.5, 1973.5 ] ] },
-{ "id": 463, "submitted_by": "xXLInkster17Xx", "name": "Little Big Planet","description": "r/LittleBigPlanet was created by several dedicated members of the Little Big Planet Union, it includes the well-known Little Big Planet tree, a sackboy face to the right and the r/LittleBigPlanet sub-reddit text below the tree which used to read \"My Little Big Planet\" but was changed due to pixel limitations.", "website": "https://littlebigplanet.com/", "subreddit": "r/LittleBigPlanet", "center": [ 1046.5, 385.5 ], "path": [ [ 1064.5, 400.5 ], [ 1033.5, 400.5 ], [ 1033.5, 366.5 ], [ 1051.5, 366.5 ], [ 1051.5, 373.5 ], [ 1062.5, 373.5 ], [ 1062.5, 379.5 ], [ 1051.5, 379.5 ], [ 1051.5, 387.5 ], [ 1064.5, 387.5 ], [ 1064.5, 400.5 ] ] },
+{ "id": 463,"name": "Flag of Vietnam","description": "Vietnam is a long, narrow nation shaped like the letter S located at the eastern edge of South East Asia's mainland which has a population about 96M people. With over thousands of years of fighting against the colonization of superpowers and imperialism in the past.","website": "","subreddit": "/r/Vietnam","center": [ 437.5,1690.5 ],"path": [ [ 447.5,1687.5 ], [ 414.5,1676.5 ], [ 461.5,1676.5 ], [ 460.5,1704.5 ], [ 413.5,1703.5 ], [ 414.5,1676.5 ], [ 414.5,1676.5 ], [ 414.5,1676.5 ], [ 414.5,1676.5 ], [ 414.5,1676.5 ] ] },
```

Comment:
Why did you remove Little Big Planet?

Reply:
how can i also make a post request for our sign?

Reply:
If you want to add your item to place-atlas take a look at [this post](https://www.reddit.com/r/placeAtlas2/comments/tu203o/how_to_contribute/) in the subreddit.
This isn't exactly the place for asking questions, so further questions should stay there aswell.